extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -65,9 +84,25 @@ export class AmpSelector extends AMP.BaseElement {
       this.element.setAttribute('aria-disabled', 'true');
     }
 
+    const kbSupportMode = this.element.getAttribute('keyboard-support');
```

Comment:
we should guard this feature by an experiment and also create issue for validator work required.